src/components/Calculator.js

```diff
@@ -1,43 +1,49 @@
-            <td className="button1">%</td>
-            <td className="button2"><span>&#247;</span></td>
-          </tr>
-          <tr>
-            <td className="button1">7</td>
-            <td className="button1">8</td>
-            <td className="button1">9</td>
-            <td className="button2">x</td>
-          </tr>
-          <tr className="tr">
-            <td className="button1">4</td>
-            <td className="button1">5</td>
-            <td className="button1">6</td>
-            <td className="button2">-</td>
-          </tr>
-          <tr className="tr">
-            <td className="button1">1</td>
-            <td className="button1">2</td>
-            <td className="button1">3</td>
-            <td className="button2">+</td>
-          </tr>
-          <tr className="tr">
-            <td className="button1" colSpan="2">0</td>
-            <td className="button1">.</td>
-            <td className="button2">=</td>
-          </tr>
-        </table>
-      </div>
+      <>
+        <div className="calculator-container">
+          <div className="calc-input"><span className="result">{next || total || 0}</span></div>
+          <div className="buttons">
+            <button type="button" className="bg-1" onClick={this.handleClick}>AC</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>+/-</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>%</button>
+            <button type="button" className="bg-1 active" onClick={this.handleClick}><span>&#247;</span></button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>7</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>8</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>9</button>
+            <button type="button" className="bg-1 active" onClick={this.handleClick}>x</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>4</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>5</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>6</button>
+            <button type="button" className="bg-1 active" onClick={this.handleClick}>-</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>1</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>2</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>3</button>
+            <button type="button" className="bg-1 active" onClick={this.handleClick}>+</button>
+            <button type="button" className="zero" onClick={this.handleClick}>0</button>
+            <button type="button" className="bg-1" onClick={this.handleClick}>.</button>
+            <button type="button" className="bg-1 active" onClick={this.handleClick}>=</button>
```

Comment:
[OPTIONAL]

- I think it will be better if you create a reusable button component that you will use throughout your app (one of the beauties of React 😄) instead of writing too many buttons which means repeating yourself. Please consider this 👍🏽